extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -43,21 +44,31 @@ export class AmpExperiment extends AMP.BaseElement {
 
     const config = this.getConfig_();
     const results = Object.create(null);
+    const override = this.getVariantOverride_();
     const variants = Object.keys(config).map(experimentName => {
+      if (override[experimentName]) {
+        results[experimentName] = override[experimentName];
+        return Promise.resolve();
```

Comment:
No need to return  a Promise here, `Promise.all` will cast itself as necessary (and sometimes optimize by not creating an instance at all).